src/Castle.Facilities.AspNetCore/CastleWindsorRegistrationExtensions.cs

```diff
@@ -29,44 +32,172 @@ namespace Castle.Facilities.AspNetCore
 	using Microsoft.AspNetCore.Razor.TagHelpers;
 	using Microsoft.Extensions.DependencyInjection;
 
+	/// <summary>
+	/// For overriding default registration and lifestyles behaviour
+	/// </summary>
+	public class CastleWindsorRegistrationOptions
```

Comment:
Should these classes also drop the "Castle" bit?

Reply:
Yup. Definitely.